.github/workflows/merge-to-main.js

```diff
@@ -0,0 +1,77 @@
+const {
+  slackNotification,
+  getLocalConfigs,
+  isWithinRCP,
+  pulls: { addLabels, addFiles, getChecks, getReviews },
+} = require('./helpers.js');
+
+// Run from the root of the project for local testing: node --env-file=.env .github/workflows/merge-to-main.js
+const PR_TITLE = '[Release] Stage to Main';
+const STAGE = 'stage';
+const PROD = 'main';
+
+let github, owner, repo;
+
+const getStageToMainPR = () =>
+  github.rest.pulls
+    .list({ owner, repo, state: 'open', base: PROD, head: STAGE })
+    .then(({ data } = {}) => data.find(({ title } = {}) => title === PR_TITLE))
+    .then((pr) => pr && addLabels({ pr, github, owner, repo }));
+
+const workingHours = () => {
+  const now = new Date();
+  const day = now.getUTCDay();
+  const hour = now.getUTCHours();
+  const isSunday = day === 0;
+  const isSaturday = day === 6;
+  const isFriday = day === 5;
+  return hour >= 8 && hour <= 20 && !isFriday && !isSaturday && !isSunday;
```

Comment:
This sounds like a good start for IST/Europe. Do you see this expanding to the US timezone as well at one point?

Reply:
8PM UTC would be:
- 4PM east coast (New York)
- 2PM (Utah)
- 1PM (San Jose)

I tried to account for an overlap of all (US) time zones without stretching them out too aggressively to give us good coverage in terms of available engineers. 

If you have any better suggestions, I'd gladly implement them - otherwise I'd recommend we start with trying to merge "earlier" into the day